job_definitions/notify_suppliers_of_framework_application_events.yml

```diff
@@ -75,12 +65,14 @@
             FLAGS="$FLAGS --dry-run"
           fi
 
+          EMAIL_UUID="$(echo $EMAIL_TYPE | cut -d ':' -f 2 -)"
+
           docker run -e \
-            DM_DATA_API_TOKEN_{{ environment|upper }} \
-            digitalmarketplace/scripts scripts/notify-suppliers-of-framework-application-event.py \
-            '{{ environment }}' \
-            $FRAMEWORK_SLUG \
-            $NOTIFY_API_TOKEN \
-            '{{ notify_template_map[EMAIL_TYPE] }}' \
+            "DM_DATA_API_TOKEN_{{ environment|upper }}" \
+            digitalmarketplace/scripts scripts/framework-applications/notify-suppliers-of-framework-application-event.py \
+            "{{ environment }}" \
+            "$FRAMEWORK_SLUG" \
+            "$NOTIFY_API_TOKEN" \
+            "$EMAIL_UUID" \
```

Comment:
Can we rename this to be `$NOTIFY_TEMPLATE_ID` to be consistent with other jobs?